charts/streams-app-cleanup-job/README.md

```diff
@@ -0,0 +1,20 @@
+# streams-app cleanup helm chart
+
+This chart can be used to deploy a cleanup job for your Kafka Streams app developed using streams-bootstrap.
+
```

Comment:
Maybe you can repeat here, which steps cleanup involves. Also, you could elaborate on the intended use: should the app deployment be deleted before running the job? Which parts of the config must be the same as in the cleaned app?

Reply:
I added a note. However, I won't repeat what the clean up does. That is already part of the global README and this is just about deploying it to Kubernetes